README.md

```diff
@@ -64,11 +64,47 @@ At the core of n2y are a set of python classes that represent the various parts
 
 | Notion Object Type | Description |
 | --- | --- |
+| Block | A bit of content within a Page |
```

Comment:
@sHermanGriffiths this table, which includes the Notion Object Types, should _not_ include all of the individual block types. There is a table further down in the readme for the block types that contains some of the block types already.

Also, the descriptions should be empty unless there is something non-obvious about the behavior. Thus, comments like "A notion block of an image" should be omitted. This was the intent of the GitHub issue that stated "Only add information that is not obvious; e.g., if an paragraph block just makes a paragraph in markdown, we don't need to add any comment about it.". I think this will make the table more useful if it only contains the non-obvious stuff.